rules/windows/process_creation/proc_creation_win_more_vbc_lummac.yml

```
@@ -0,0 +1,28 @@
+title: Detect The Execution Of More.com And Vbc.exe Related to Lummac Stealer
+id: 19b3806e-46f2-4b4c-9337-e3d8653245ea
+status: test
```

```suggestion
status: experimental
```
see https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq-rule-convention.md#status